### PR TITLE
Task/13 Created title component

### PR DESCRIPTION
**Pull request by Zalezny**

### CODE REVIEW

**Comment by msarelo** [comment]
The implementation is good. But let's try to do that in simpler way. Please try to use only style :), hints:
- drawableStart
- drawablePadding
- you can set size (<size></size>) inside shape

 or just extends in H2View AppCompatTextView and ImageView on the left side of text.

File: app/src/main/java/com/safetyheads/akademiaandroida/H2View.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    defStyleAttr: Int = 0
+) : LinearLayout(context, attrs, defStyleAttr) {
+
+    private val attributes: TypedArray = context.obtainStyledAttributes(attrs, R.styleable.H2View)
+    private val view: View = View.inflate(context, R.layout.h2_view, this)
+
+    init {
+        val textH2 = view.findViewById<TextView>(R.id.text_view_h2)
+        textH2.text = attributes.getString(R.styleable.H2View_text)
+
```
Comment (przemek19980102) [inline app/src/main/java/com/safetyheads/akademiaandroida/H2View.kt:30]: Remove empty lines here

File: app/src/main/res/drawable/line_2422.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (przemek19980102) [inline app/src/main/res/drawable/line_2422.xml:2]: Name of the file should be better, for example h2_view_line

File: app/src/main/res/layout/h2_view.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (przemek19980102) [inline app/src/main/res/layout/h2_view.xml:2]: please rename it to view_h2

**Review by przemek19980102 — CHANGES_REQUESTED** [review]
Some small changes but otherwise looks ok :)

**Review by przemek19980102 — APPROVED** [review]

**Pull request merged**